tests/boot/with_mcumgr/testcase.yaml

```diff
@@ -0,0 +1,30 @@
+common:
```

Comment:
please add
`slow: true`
Since those tests require 600s to pass they fall into "slow" category which shouldn't be executed by twister by default.